src/vs/editor/browser/editorBrowser.ts

```diff
@@ -332,7 +332,8 @@ export interface IOverviewRuler {
  * @internal
  */
 export interface IEditorAriaOptions {
-	activeDescendant: string | undefined;
+	activeDescendant?: string | undefined;
```

Comment:
I do not like that these attributes are both optional and can be `undefined`.
I would just go with one approach and stick with that. Either they are optional or can be `undefined`